### DISCUSSION TITLE
Why?

### DISCUSSION THREAD

**Discussion by Ragemalik** [vilhelmprytz/pterodactyl-installer#197] [General]
Hey

Quick question I was wondering if this script has the same features same bug fixes same overall stuff like basically the same if it's not and has more features what are they? Also when I make a database through the Web gui will It create easily and will I get a link so I can login to the database?.

Thank you.

**Comment by sinjs** [comment]
The script installs pterodactyl from the official GitHub repository, so it has the same features as manually installed Pterodactyl .

> Also when I make a database through the Web gui will It create easily and will I get a link so I can login to the database?.

No, you won't. You will get MySQL username, database and password so you can login with another database client / use them for something else.

**Comment by vilhelmprytz** [comment]
What are you comparing the script to? This is not an alternative to Pterodactyl, this is a script for installing the official Pterodactyl panel. This is not an issue with the script, so I've converted this to a discussion thread instead 🙂.
